packages/flex-plugins-api-toolkit/src/scripts/diff.ts

```diff
@@ -17,7 +17,11 @@ export interface DiffOption {
   newIdentifier: string;
 }
 
-export type Diff = ConfigurationsDiff;
+export interface Diff extends ConfigurationsDiff {
+  oldSid: string;
+  newSid: string;
+  activeSid: string | undefined | null;
```

Comment:
would `activeSid?: string` not work for this case?

Reply:
No, the key would always exist, it just may be null or undefined.